src-tauri/src/platform_specific/linux.rs

```diff
@@ -3,6 +3,79 @@
 use regex::Regex;
 use std::process::Command;
 
+fn get_proton_dir() -> Option<String> {
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+
+    Some(compat_dir)
+}
+
+/// Downloads and installs NS proton
+/// Assumes Steam install
+pub fn install_ns_proton() -> Result<(), thermite::prelude::ThermiteError> {
+    // Get latest NorthstarProton release
+    let latest = thermite::core::latest_release()?;
+
+    let temp_dir = std::env::temp_dir();
+    let path = format!("{}/nsproton-{}.zip", temp_dir.display(), latest);
+    let archive = std::fs::File::open(path.clone()).unwrap();
```

Comment:
So trying to run _Install NSProton_ I get a panic here:

```
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/platform_specific/linux.rs:21:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Yep, turned out I bunged up in the migration to the libthermite change.
It should be `File::create().unwrap_or(File::open())`